### PR TITLE
Idx 2805 restructure format of repo list

### PR DESCRIPTION
**Pull request by cgundy**
Because we will need to capture more information on public repositories, I'm converting the list to a json file. For now we are still losing the old list, the next step will be to migrate it.

### CODE REVIEW

**Comment by levshaATdfinity** [comment]
Having two list will confuse make changes more complicated. Why not prioritize cla bot to workflow migration instead to have all the code in the single place?

File: public-repositories/published-repositories.json [hunk 1]
```diff
@@ -0,0 +1,310 @@
+[
```
Comment (levshaATdfinity) [inline public-repositories/published-repositories.json:1]: Is it better than separate file per repo?

File: public-repositories/published-repositories.json [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+    {
+        "name": "ICRC",
+        "email": "",
+        "organization": "dfinity",
+        "critical": false,
```
Comment (levshaATdfinity) [inline public-repositories/published-repositories.json:6]: This needs to be clearly explained what does this field mean.

File: public-repositories/published-repositories.json [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+        "external-contributions": true
+    },
+    {
+        "name": "motoko",
+        "email": "",
+        "organization": "dfinity",
+        "critical": false,
```
Comment (levshaATdfinity) [inline public-repositories/published-repositories.json:202]: Is it just set to false for everything?

File: public-repositories/published-repositories.json (hunk 2 above)
Comment (cgundy) [reply public-repositories/published-repositories.json:6]: > Having two lists will confuse make changes more complicated. Why not prioritize cla bot to workflow migration instead to have all the code in the single place?

I'm okay with prioritizing the cla bot migration before this. Will address the comments when I pick this up again.

**Comment by cgundy** [comment]
> Having two lists will confuse make changes more complicated. Why not prioritize cla bot to workflow migration instead to have all the code in the single place?

I'm okay with prioritizing the cla bot migration before this. Will address the comments when I pick this up again.

**Pull request closed without merging**